packages/mappings/src/tier-mapping.ts

```diff
@@ -1,12 +1,12 @@
 import { type Tier } from '@repo/database';
 
-export const maxUsersPerTier: Record<Tier, { maxUsers: number }> = {
-  Launch: { maxUsers: 1 },
-  Build: { maxUsers: 4 },
-  Grow: { maxUsers: 10 },
-  Scale: { maxUsers: 100 },
+export const tierConstraints: Record<Tier, { maxUsers: number, maxRecency: number }> = {
+  Launch: { maxUsers: 1, maxRecency: 7 },
+  Build: { maxUsers: 4, maxRecency: 90 },
+  Grow: { maxUsers: 10, maxRecency: 365 },
+  Scale: { maxUsers: 100, maxRecency: Infinity },
```

Comment:
```suggestion
  Scale: { maxUsers: 100, maxRecency: 365 },
```
Let's put 365, infinite is a dangerous setting